kanban/models/issue.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from kanban.models.project import Project
+from user_account.models import UserAccount
+
+
+class Issue(models.Model):
+    TODO = "todo"
+    IN_PROGRESS = "in progress"
+    REVIEW = "review"
+    DONE = "done"
+    STATUS = [
+        (TODO, "TODO"),
+        (IN_PROGRESS, "IN_PROGRESS"),
+        (REVIEW, "REVIEW"),
+        (DONE, "DONE"),
+    ]
+
+    title = models.CharField(max_length=30, unique=True)
```

Comment:
Why does title have to be unique?

Reply:
Because when I'm assigning issue to the project I want to be able to add an existing one by title or create it if issue with such title is not found (kanban.api.views.project , line 63). I could use id, but I couldn't use create method then (it would be strange to force user to choose id for created object I quess, wouldn't it?). 
Or should I not allow creating object at all, just assinging existing one?

Reply:
You should pretty much always assigns stuff by id. It's kinda ok in terms of inviting users to project, that you go by email. But all the other stuff should be pretty much always id.

If you have different serializers for issue list, issue view and assign, then only keep ability to create only on the "issue view" serializer